task_sdk/tests/conftest.py

```diff
@@ -23,4 +23,4 @@
 
 @pytest.hookimpl(tryfirst=True)
 def pytest_configure(config: pytest.Config) -> None:
-    config.inicfg["airflow_deprecations_ignore"] = ()
+    config.inicfg["airflow_deprecations_ignore"] = ()  # type: ignore[assignment,operator]
```

Comment:
Instead of assigning to a tuple, I did think of assigning to an empty string. But, I don't know if that's correct.

Reply:
I changed it to string